core/src/com/unciv/ui/screens/basescreen/TutorialController.kt

```diff
@@ -29,18 +30,21 @@ class TutorialController(screen: BaseScreen) {
         // static to allow use from TutorialTranslationTests
         fun loadTutorialsFromJson(includeMods: Boolean = true): LinkedHashMap<String, Tutorial> {
             val result = linkedMapOf<String, Tutorial>()
-            for (path in tutorialFiles(includeMods)) {
-                json().fromJsonFile(Array<Tutorial>::class.java, path)
+            for (file in tutorialFiles(includeMods)) {
+                json().fromJsonFile(Array<Tutorial>::class.java, file)
                     .associateByTo(result) { it.name }
             }
             return result
         }
-        private fun tutorialFiles(includeMods: Boolean) = sequence<String> {
-            yield("jsons/Tutorials.json")
+
+        private fun tutorialFiles(includeMods: Boolean) = sequence<FileHandle> {
+            yield(Gdx.files.internal("jsons/Tutorials.json"))
             if (!includeMods) return@sequence
-            val mods = UncivGame.Current.gameInfo?.ruleset?.mods ?: return@sequence
-            val names = mods.asSequence().map { "mods/$it/jsons/Tutorials.json" }
-            yieldAll(names.filter { Gdx.files.local(it).exists() })
+            val mods = UncivGame.Current.gameInfo?.ruleset?.mods
+                ?: return@sequence
+            val files = mods.asSequence()
+                .map { Gdx.files.local("mods/$it/jsons/Tutorials.json") }
```

Comment:
I mean if we already have a sequence, add the .filter exists here, rather than in the yieldAll

Reply:
![image](https://github.com/yairm210/Unciv/assets/8366208/c7f3e142-e11d-4b23-bac8-baef6f57cd24)
I still get them, sounds like something on your machine